### DISCUSSION TITLE
Email digest

### DISCUSSION THREAD

**Discussion by joebudi** [community/community#4075] [Other Feature Feedback, Questions, & Ideas]
As a maintainer, I would like the option to circulate a weekly email digest to subscribers of the Budibase Discussions channel, as this would keep them updated on community issues/news.

**Comment by j-peeters** [comment]
Yes, a weekly digest function would be great. I would also want to subscribe to such a digest without the admin needing to circulate it to me. So please make this a user based chose.

**Reply by thomas-schuster** [reply]
Should the client side digest to discussion be a separate request?

**Reply by joebudi** [reply]
I think it should.
